jawab.txt

```diff
@@ -0,0 +1,5 @@
+Pertama buat 1 branch realese untuk dedline minggu depan.
```

Comment:
I assume we create this release branch, from master branch,

Reply:
iya pak, sebelumnya jawaban saya sudah saya lakukan pembaharuan